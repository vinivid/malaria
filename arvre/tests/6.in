i Yara
i Xena
i Ze
i Carlos
i Nadia
