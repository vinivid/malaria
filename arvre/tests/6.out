digraph {
"0 Yara" ->"1 Xena" 
"0 Yara" ->"2 Ze" 
"1 Xena" ->"3 Carlos" 
"3 Carlos" ->"8 Nadia" 
}

digraph {
"0 Xena" ->"1 Carlos" 
"0 Xena" ->"2 Yara" 
"1 Carlos" ->"4 Nadia" 
"2 Yara" ->"6 Ze" 
}

digraph {
"0 Yara" ->"1 Nadia" 
"0 Yara" ->"2 Ze" 
"1 Nadia" ->"3 Carlos" 
"1 Nadia" ->"4 Xena" 
}

